src/_cffi_src/openssl/dh.py

```diff
@@ -114,4 +114,100 @@
     return 1;
 }
 #endif
+
+#if CRYPTOGRAPHY_OPENSSL_LESS_THAN_110 || defined(LIBRESSL_VERSION_NUMBER)
+/*-
+ * Check that p is a safe prime and
+ * if g is 2, 3 or 5, check that it is a suitable generator
+ * where
+ * for 2, p mod 24 == 11
+ * for 3, p mod 12 == 5
+ * for 5, p mod 10 == 3 or 7
+ * should hold.
+ */
+
+int Cryptography_DH_check(const DH *dh, int *ret)
```

Comment:
[GitHub says 1.1.0pre6](https://github.com/openssl/openssl/commit/748e85308ef4f3e672975b3604ea2d76424fa404) - not sure which letter release that is.  How can I find out?